composer.json

```diff
@@ -0,0 +1,221 @@
+{
+    "name": "linnovate/openideal-composer",
```

Comment:
Should we change to "linnovate/openideal-on-platformsh"?

Reply:
I'm not sure if we need to change it. It's just a repository that fetches changes from linnovate/openideal-composer with the predefined composer.lock file.